content/guides/translation-connection/step3-how-to-build-your-connector.md

```diff
@@ -183,11 +183,20 @@ With these new attributes, your connector is now ready to go! Let's see how it'l
 
 ### How it works with your connector?
 
-**Find products with the "translation queued" status enabled**
+**Pull or Push PIM information data?**
+
+Now that everything is ready in your PIM, your connector is now ready to retrieve the PIM data to be translated.
+
+From this point, you have 2 possibilities:
```

Comment:
I prefer avoiding this type of communication. 
We should recommend using the REST API and tell them that they can also use the Events API in addition to synchronization through REST API if they want to avoid pulling data too often.